mne_bids/write.py

```diff
@@ -821,6 +871,29 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         inferred from the stim channel using `mne.find_events`.
     event_id : dict | None
         The event id dict used to create a 'trial_type' column in events.tsv
+    anonymize : dict | None
+        If a dictionary is passed, data will be anonymized; identifying data
+        structures such as study date and time will be changed.
+        `daysback` is a required argument and `keep_his` is optional, these
+        arguments are passed to :func:`mne.io.anonymize_info`.
+
+        `daysback` : int
+            Number of days to move back the date. NB: To keep relative dates
+            for a subject use the same `daysback`. According to BIDS
+            specificiations, the number of days back must be great enough
+            that the date is before 1900 thus `daysback` will be subtracted
+            from Dec 31, 1924.
+
+        `keep_his` : bool
+            If True his_id of subject_info will NOT be overwritten.
+            Defaults to False.
+
+    convert : bool | None
```

Comment:
can we move this to another PR @alexrockhill ? I need to think a bit on the question of `None`

Reply:
Sure, doing that now

https://gist.github.com/alexrockhill/eda1562e8ed22b933cab730c9e9a5851